query/src/test/scala/filodb/query/exec/rangefn/AbsentFunctionSpec.scala

```diff
@@ -100,7 +100,11 @@ class AbsentFunctionSpec extends AnyFunSpec with Matchers with ScalaFutures with
     val result = resultObs.toListL.runToFuture.futureValue
     val keys = result.map(_.key.labelValues)
     val rows = result.flatMap(_.rows.map(_.getDouble(1)).toList)
-    rows.isEmpty shouldEqual true
```

Comment:
Could you update the name of this test? (Awkwardly adding the comment here because I can't apply it above)